release.sh

```diff
@@ -1,29 +1,48 @@
 #!/bin/sh
 
```

Comment:
Seems like a script where we should exit early and loudly on failure.
```suggestion

set -eu

```

Reply:
added in the latest commit